### ISSUE TITLE
'alt text' doesn't work

### ISSUE THREAD

**Issue by Kaiux** [hexojs/hexo#2887]
```text
{% img [class names] /path/to/image [width] [height] [title text [alt text]] %}
```
I use 'Tag Plugins' to define the size of image，but how to define 'alt text'. I have tried many time but it dose not work. I think there should an separator between 'title text' and 'alt text'.

**Comment by tcrowe** [comment]
It looks like `alt` is nested inside `title`. Is that allowed in the template?

Which template engine?

**Comment by Kaiux** [comment]
I find the code as follow
```javascript
var rMeta = /["']?([^"']+)?["']?\s*["']?([^"']+)?["']?/;
     ……
// Find image title and alt
    if (meta && rMeta.test(meta)) {
      var match = meta.match(rMeta);
      title = match[1];
      alt = match[2];
}
```
but, how does the regex works ? it seems like match all the string and i found it would match twice.

**Comment by Kaiux** [comment]
The question is solved. [Reference](https://stackoverflow.com/questions/47665964/hexo-alt-text-doesnt-work)
